server/modules/instantiation/jobs.js

```diff
@@ -329,7 +329,14 @@ module.exports = function (config, portProvider, interpolationHelper, buildInsta
                 _.each(
                     buildInstance.componentInstances,
                     (componentInstance, componentId) => {
-                        buildInstance.addEnvironmentalVariable(`FEAT__PATH__${componentId.toUpperCase()}`, componentInstance.fullPath);
+                        buildInstance.addEnvironmentalVariable(
+                            `FEAT__BUILD_PATH__${componentId.toUpperCase()}`,
+                            componentInstance.fullPath
+                        );
+                        buildInstance.addEnvironmentalVariable(
+                            `FEAT__VOLUME_PATH__${componentId.toUpperCase()}`,
+                            componentInstance.fullPath.replace(/^\/app\//, '/home/malef/Development/Feat/') // TODO Need to provide absolute path on host, should be passed via config or environment.
```

Comment:
This is a place that still needs to be adjusted manually. Working on moving this to config file.